tests/simulation/framework/test_entities_v2.py

```diff
@@ -858,3 +859,67 @@ def test_corner_cases_for_entity_registry_thoroughness(my_cylinder1, my_volume_m
         my_volume_mesh_with_interface["innerZone"]
     )
     assert my_param.private_attribute_asset_cache.asset_entity_registry.entity_count() == 11
+
+
+def compare_boxes(box1, box2):
+    return (
+        np.isclose(np.linalg.norm(np.cross(box1.axis_of_rotation, box2.axis_of_rotation)), 0)
+        and np.isclose(
+            np.mod(box1.angle_of_rotation.value, 2 * np.pi),
+            np.mod(box2.angle_of_rotation.value, 2 * np.pi),
+        )
+        and np.all(np.isclose(box1.center.value, box2.center.value))
+        and np.all(np.isclose(box1.size.value, box2.size.value))
+        and np.all(
+            np.isclose(
+                np.asarray(box1.private_attribute_input_cache.axes, dtype=float),
+                np.asarray(box2.private_attribute_input_cache.axes, dtype=float),
+            )
+        )
+    )
+
+
+def test_box_creation():
+    box1 = Box(
+        name="box1",
+        center=[0, 0, 0] * u.m,
+        size=[1, 1, 1] * u.m,
```

Comment:
Interesting. I thought list is not supported.

Reply:
Can we change and use tuple to be consistent?

Reply:
Can we add a 0 rotation test, x to y and y to x, 90 degree test? You can use update() to update only the part you need to change for less coding.